KAS-2660 Make button of date published to agendapoint

TaneeG opened the pull request.

Commit:
KAS-2660 Make button of date published to agendapoint

Commit:
Merge branch 'development' into feature/KAS-2660_van_publicatie_naar_MR

MikiDi commented on app/components/publications/publication-sidebar.hbs:
```diff
@@ -119,9 +119,14 @@
-            <span class="auk-u-muted">
-              {{moment-format this.treatment.startDate "DD-MM-YYYY"}}
-            </span>
+            <Auk::ButtonLink {{on "click" this.navigateToAgendaItem }}>
```
Idealiter is dit toch een echte link (die je in een nieuwe tab kan openen). Zie bv https://github.com/kanselarij-vlaanderen/kaleidos-frontend/blob/97498b4479f3a4c63d6522f5d30e9733776dded8/app/components/agenda/agendaitem/agendaitem-case/agendaitem-titles.hbs#L21 voor een voorbeeld van een link waarvan onderdelen async geladen moeten worden.

Commit:
KAS-2660 change button type

erikap committed:
Use Auk::ButtonLink to navigate from publication to agendaitem

erikap committed:
Remove awaits from template

In contrast to JS files ember-data relations must not be explicitely
awaited in template files

erikap committed:
Remove unused route service injection
